sponsor-dapp-v2/src/lib/contracts.js

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect } from "react";
+import { drizzleReactHooks } from "drizzle-react";
+
+function useDynamicallyAddedContract(contractAddress, abi) {
+  const { drizzle } = drizzleReactHooks.useDrizzle();
+  const { contract } = drizzleReactHooks.useDrizzleState(drizzleState => ({
```

Comment:
Should this be recomputed whenever contractAddress changes?

Reply:
Done.